### PR TITLE
Enable the costume tab by default in SpriteLab

### PR DESCRIPTION
**Pull request by jmkulwik**
https://codedotorg.atlassian.net/browse/STAR-191
https://codedotorg.atlassian.net/browse/STAR-375
Waiting for after Thursday May 2nd's DTP to complete the PR

### CODE REVIEW

File: apps/src/gamelab/blocks.js [hunk 1]
```diff
@@ -154,7 +153,7 @@ const customInputTypes = {
   costumePicker: {
     addInput(blockly, block, inputConfig, currentInputRow) {
       let buttons;
-      if (experiments.isEnabled('sprite-costumes')) {
+      if (getStore().getState().pageConstants.showAnimationMode) {
```
Comment (epeach) [inline apps/src/gamelab/blocks.js:156]: Limit to just Spritelab?

File: dashboard/config/scripts/levels/New Sprite Lab Project.level [hunk 2]
```diff
@@ -30,9 +30,9 @@
     "hide_share_and_remix": "false",
     "disable_if_else_editing": "false",
     "free_play": "false",
-    "hide_animation_mode": "true",
+    "hide_animation_mode": "false",
```
Comment (epeach) [inline dashboard/config/scripts/levels/New Sprite Lab Project.level:33]: Do we need to do this across all the levels for which the curriculum team wants to have the costume tab?

File: apps/src/gamelab/GameLab.js [hunk 3]
```diff
@@ -379,10 +374,10 @@ GameLab.prototype.init = function(config) {
     showDebugWatch:
       config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
     showDebugSlider: experiments.isEnabled('showDebugSlider'),
-    showAnimationMode: !config.level.hideAnimationMode || showCostumeTab,
+    showAnimationMode: !config.level.hideAnimationMode,
     startInAnimationTab: config.level.startInAnimationTab,
     allAnimationsSingleFrame:
-      config.level.allAnimationsSingleFrame || showCostumeTab,
+      config.level.allAnimationsSingleFrame || this.studioApp_.isUsingBlockly(),
```
Comment (epeach) [inline apps/src/gamelab/GameLab.js:380]: If we are setting this to true in the level config, do we need to have `this.studioApp_.isUsingBlockly()`?

Comment (jmkulwik) [reply apps/src/gamelab/GameLab.js:380]: There is a potential that level builders could accidentally set allAnimationsSingleFrame to false in a sprite lab level, but we want to force it to always be true in sprite lab.

File: dashboard/config/scripts/levels/New Sprite Lab Project.level (hunk 2 above)
Comment (jmkulwik) [reply dashboard/config/scripts/levels/New Sprite Lab Project.level:33]: Already been added :)

File: apps/src/gamelab/blocks.js (hunk 1 above)
Comment (jmkulwik) [reply apps/src/gamelab/blocks.js:156]: As opposed to Gamelab? This should already be only Spritelab because it is blockly blocks.

**Review by epeach — APPROVED** [review]
jazzybird
LGTM

**Pull request merged**